src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/validators.py

```diff
@@ -24,11 +24,6 @@ def _final_validator_impl(cmd, namespace):
         for validator in validators:
             validator(namespace)
 
-        if namespace.sku.tier or namespace.sku.capacity:
```

Comment:
Why are these being removed?

Reply:
Skuname wasn't a required field earlier, for creating a server; But, now it is. So, this validation is no longer applicable.